lib/charms/layer/jenkins/plugins.py

```diff
@@ -11,6 +12,15 @@
 class Plugins(object):
     """Manage Jenkins plugins."""
 
+    def __init__(self):
+        plugins_site = hookenv.config()["plugins-site"]
+        try:
+            update_center = plugins_site + "/update-center.json"
+            urllib.request.urlopen(update_center)
+            self.update_center = UpdateCenter(uc_url=update_center)
+        except urllib.error.HTTPError:
```

Comment:
If I'm understanding this correctly it's going to silently fall back to the default if it can't reach the plugins_site that's been specified in juju config? That doesn't seem like something we'd want, I'd assume we'd want to raise an error and/or block juju status if we can't reach the site we've configured.

Reply:
Addressed that and now the charm will block if the file is unreachable.